.github/workflows/container-image.yml

```diff
@@ -18,6 +21,10 @@ on:
         required: true
         type: string
         description: 'URL to the Icinga documentation for this project.'
+      container_readme_filepath:
+        required: false
+        type: string
+        description: 'Path to the README file to sync with Docker Hub. Defaults to the first For-Container.md file found in the ./doc/ directory.'
```

Comment:
> first For-Container.md

The implementation uses `find` and that guarantees no ordering, so if there were multiple files, one would basically be picked at random.